src/main/java/org/marketplace/configuration/DataLoader.java

```diff
@@ -0,0 +1,31 @@
+package org.marketplace.configuration;
+
+import org.marketplace.models.User;
+import org.marketplace.repositories.UserManagementRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataLoader implements ApplicationListener<ApplicationReadyEvent> {
+
+    @Autowired
+    private UserManagementRepository userRepository;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+    @Override
+    public void onApplicationEvent(ApplicationReadyEvent event) {
+        if (userRepository.count() == 0) {
+            User user = new User();
+            user.setEmail("user1@test.com");
```

Comment:
i could imagine that we could hide default user and pass behind linux env properties and autowire them into runtime. just to not stop the work shall i fill out an issue?

Reply:
Good idea, I have filled out an issue #22 and will get to work with that.